package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "voting-rewards",
-  "version": "0.0.1",
+  "version": "0.0.3",
```

Comment:
The typical order of the versioning is `major.minor.bugfix`, where major is something that's a breaking change. 

So this PR likely want's the minor version number bumping, not the bugfix one.

Reply:
yesterday i had to fix a small bug for a testnet deployment so it should be fine!

Reply:
Ah fair enough. Though this was for this whole PR, nevermind, carry on!